src/app/containers/SwapFormContainer/index.tsx

```diff
@@ -75,8 +75,16 @@ export const SwapFormContainer: React.FC = () => {
       cancelDataRequest.current = c;
     });
     axios
-      .get(graphWrapperUrl[currentChainId] + 'cmc/summary?extra=true', {
-        cancelToken,
+      .get(graphWrapperUrl[currentChainId] + 'cmc/summary', {
+        params: {
```

Comment:
```suggestion
      .get(graphWrapperUrl[currentChainId] + 'cmc/summary', {
        cancelToken,
        params: {
```

cancelToken is unused and the warning is causing build to fail, add it back in